lambdas/service/app.py

```diff
@@ -86,7 +86,77 @@
+            querying metadata associated with experimental and analysis data
+            stored in the
+            [data store (DSS)](https://github.com/HumanCellAtlas/data-store).
+            Azul serves as the backend for the
+            [data-browser](https://github.com/DataBiosphere/data-browser).
+
+            Azul aggregates metadata into several different indices:
+             - [files](#operations-Repository-get_repository_files)
+             - [samples](#operations-Repository-get_repository_samples)
+             - [bundles](#operations-Repository-get_repository_bundles)
+
+            which are all stored in respective Elasticsearch indices.
+
+            Azul provides the ability to download relevant data via the
+            [Manifests](#/Manifests) endpoints. After refining a search with the
+            [Repository](#/Repository) endpoints, these can be used to download
+            a manifest of the relevant files. The manifest can then be used by
+            the [DCP CLI](https://github.com/HumanCellAtlas/dcp-cli) to download
+            the listed files.
+
+            Azul also provides a
+            [summary](#operations-Repository-get_repository_summary) view of
+            indexed data.
+
+            ## Data model
+            Each index, when queried returns a JSON array of _entities_. An
+            entity is comprised of a handful of different kinds of _inner
+            entities_. Inner entities contains concrete metadata fields. Each
+            inner entity corresponds to a metadata object in the original
+            metadata graph.
+
+            To make this concrete, suppose you had a file that is mentioned in
+            several projects. Querying the `files` index for this file will give
+            an entity that looks something like:
+
+            ```
+            {
+                "projects": [
+                    {
+                        "projectTitle": "Project One"
+                        "laboratory": ...,
+                        ...
+                    },
+                    {
+                        "projectTitle": "Project Two"
+                        "laboratory": ...,
+                        ...
+                    }
+                ],
+                "files": [
+                    {
+                        "format": "pdf",
+                        "name": "Team description.pdf",
+                        ...
+                    }
+                ]
+            }
+            ```
+            This example entity contains two kinds of inner entities (an
+            actual entity will contain more). There are the two `projects` inner
+            entities, one `files` inner entity. These inner entities contain
+            useful and relevant metadata fields extracted in a consistent way.
+            This makes searching and sorting simple.
+
+            Also notice that the `files` inner entity is a singleton. When
```

Comment:
We should probably avoid the term singleton here:

```suggestion
            Also notice that there is only one inner file entity. When
```

and so on